New manual translation process (#1041)
ministryofjustice/cla_public: docs/translation.md

You should check all URLs at point of translation to see if there is a Welsh version of the website you're linking to.

If there is no Welsh version to link to, then you should consider using a variable which is easier to maintain should the URL change.

    {% trans
        email = Element.link_same_window('mailto:civil-legal-advice@digital.justice.gov.uk', 'civil-legal-advice@digital.justice.gov.uk', **{'class': 'email'})
    %}You can contact us at {{email}}{% endtrans %}

    msgid "You can contact us at %(email)s"
    msgstr "Gallwch gysylltu â ni yn %(email)s."

## Translation quality

When updating the translation file, it is important to maintain the quality of the text.  

- Ensure that all text uses smart quotes (`’` instead of `'`).
- Ensure all abbreviations are wrapped in `<abbr>` tags.
- - If it isn't obvious from context, expand the abbreviations with their meaning by adding a `title` attribute (e.g. `<abbr title='Legal Aid Agency'>LAA</abbr>`).
- - The abbreviation doesn't change with Welsh, e.g. `<abbr title='Asiantaeth Cymorth Cyfreithiol'>LAA</abbr>`.

Ensure that both the English on the HTML page and the Welsh translations are of good quality.

> The content designer owns the content, so if you think there is poor wording or something should be rephrased, defer to him.  He will decide whether or not the Welsh needs to be redone - but often, if neither the meaning nor the emphasis of the sentence has changed, the Welsh will not need to be altered.  

Reviewer: ```suggestion
The content designer owns the content, so if you think there is poor wording or something should be rephrased, defer to them.  They will decide whether or not the Welsh needs to be redone - but often, if neither the meaning nor the emphasis of the sentence has changed, the Welsh will not need to be altered.  
```
